.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "3.6"
+install:
+  - pip install -r requirements.txt
+script:
+  - pyflakes assigner/**/*.py assigner/*.py
```

Comment:
This should just be `pyflakes assigner` -- `pyflakes` (and `pylint`) take the name of a python module to check.

Reply:
Thanks, this isn't like flake8 at all.